about/contributing_datasets.md

```diff
@@ -0,0 +1,288 @@
+
+
+You can represent this data as a graph via subject-predicate-object "triples" that describe the node and edge relationships.
+```
+USA -- typeOf ------------> Country
+USA -- name --------------> United States of America
+USA -- containedInPlace --> northamerica
+```
+
+While you can express these triples via standards like [JSON-LD](https://json-ld.org/) or [RDFa](https://en.wikipedia.org/wiki/RDFa), the Data Commons format of choice is [MCF](https://github.com/datacommonsorg/data/tree/master/docs/mcf_format.md), which is concise and human readable. Here's the translated version of the example tabular data:
+
+```
+Node: USA
+typeOf: schema:Country
+name: "United States of America"
+containedInPlace: dcid:northamerica
+
+Node: IND
+typeOf: schema:Country
+name: "India"
+containedInPlace: dcid:asia
+```
+
+The `schema:` prefix represents the schema.org namespace, and `dcid:` represents the Data Commons namespace. For more information, see [scoping and namespaces in MCF](https://github.com/datacommonsorg/data/tree/master/docs/mcf_format.md#scoping-and-namespaces).
+
+The following sections explain how to convert [statistical data into graph representations](https://github.com/datacommonsorg/data/tree/master/docs/representing_statistics.md), as well as how to leverage templating to quickly map tabular data into [instance MCF](https://github.com/datacommonsorg/data/tree/master/docs/mcf_format.md#instance-mcf).
+
+
+## Importing The COVID Tracking Project's historical state data
+
+This particular example imports the COVID Tracking Project's [historical state dataset](https://covidtracking.com/api/).
+
+The dataset contains various statistics about COVID-19 testing results, including numbers of patients hospitalized, patients in the ICU, patients on ventilators, and patient outcomes.
+
+The end goal of this example is to represent each data point as a `StatVarObservation` like this:
+
+```
+Node: Count_MedicalTest_COVID_19_Pending_geoId/04_2020-03-04
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_MedicalTest_COVID_19_Pending
+observationAbout: dcid:geoId/04
+observationDate: "2020-03-04"
+value: 24
+```
+
+For more information on the terms [`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable) and [`StatVarObservation`](https://browser.datacommons.org/kg?dcid=StatVarObservation), see ["Intro to `StatisticalVariable` and `StatVarObservation`"](https://github.com/datacommonsorg/data/tree/master/docs/representing_statistics.md).
+
+### Roadmap
+
+Importing data requires three main steps:
+
+1. Defining new `StatisticalVariable` nodes.
+2. Cleaning the CSV files.
+3. Defining a mapping from CSV to MCF (aka [Template MCF](https://github.com/datacommonsorg/data/tree/master/docs/mcf_format.md#template-mcf)).
+
+This tutorial explains each step in detail and demonstrates where you can submit artifacts for Data Commons review.
+
+## Step 1: Define new `StatisticalVariable` nodes.
```

Comment:
I believe this step should be a subsection as well? (three #), and everything below should get one added too.

Reply:
@beets I went ahead and restructured headers in this section.